src/backend/aspen/test_infra/models/pathogen_repo_config.py

```diff
@@ -1,15 +1,16 @@
 from aspen.database.models import Pathogen, PathogenRepoConfig, PublicRepository
-from aspen.test_infra.models.pathogen import pathogen_factory
+from aspen.test_infra.models.pathogen import random_pathogen_factory
+from aspen.database.models.usergroup import generate_random_id
 
 
-def setup_gisaid_and_genbank_repo_configs(async_session) -> Pathogen:
+def setup_gisaid_and_genbank_repo_configs(async_session, pathogen=None) -> Pathogen:
     repo_info = {
-        "GISAID": {"prefix": "hCoV-19"},
-        "GenBank": {"prefix": "SARS-CoV-2/human"},
+        "GISAID": {"prefix": generate_random_id(5)},
+        "GenBank": {"prefix": generate_random_id(5)},
```

Comment:
I think I can see why you want these prefixes to be random, though I'm not sure I agree. But if you do this, would you mind also fixing the tests that check if the GISAID/GenBank prefix is being dealt with properly? I  see the ones in `test_sequences` and `test_download_tree_sample_ids` are broken.